### DISCUSSION TITLE
zzh! (CC2652R) to SMLIGHT (CC2652P) coordinator

### DISCUSSION THREAD

**Discussion by et-non** [Koenkk/zigbee2mqtt#18946] [Devices]
I just replace my zzh! to smlight, which I already read on zigbee2mqtt FAQ. There doesn't require repairing.
SMIGHT, It's work well but after flash zzh! to router and joining network. It's show as coordinator.

Firmware

Coordinator:
SMLIGHT (CC2652P) > CC1352P2_CC2652P_other_coordinator_20230507

Router:
zzh! (CC2652R) > CC2652R_router_20221102

What I do wrong?

**Comment by francisp2** [comment]
Nothing. Zigbee2MQTT copied the settings from the CC2652R to the SMLIGHT, and that includes the IEEE-address. So you have to give your zzh! a new IEEE-address when flashing router firmware.

**Reply by et-non** [reply]
Thank you,

I flash zzh! with new IEEE-address it solves the problem and I flash SMLIGHT with zzh! IEEE-address, the strange behavior of coordinator in z2m MAP is now normal.

Thank you again. I'm very appreciate for your reply.

**Comment by et-non** [comment]
Today, I got warning message 

`Failed to ping 'Coordinator' (attempt 1/2, Read 0x00124b0022a54da6/8 genBasic(["zclVersion"], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":true,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 0 - 8 - 121 - 0 - 1 after 10000ms))`

`Failed to ping 'Coordinator' (attempt 2/2, Read 0x00124b0022a54da6/8 genBasic(["zclVersion"], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 0 - 8 - 122 - 0 - 1 after 10000ms))`

but I already change IEEE-address

How to solve this warning?
